lib/es-reindex.rb

```diff
@@ -19,8 +19,8 @@ def initialize(src, dst, options = {})
 
     @done = 0
 
-    @surl, @durl, @sidx, @didx = '', '', '', ''
-    [[src, @surl, @sidx], [dst, @durl, @didx]].each do |param, url, idx|
+    surl, durl, sidx, didx = '', '', '', ''
```

Comment:
I think since it's in an initializer, you'll need to use the `@`'s on the assignment line still.